OpenRA.Game/FieldLoader.cs

```diff
@@ -564,7 +564,7 @@ public static void Load(object self, MiniYaml my)
 				fli.Field.SetValue(self, val);
 			}
 
-			if (missing.Any())
+			if (missing.Count > 0)
```

Comment:
We use `Any()` on purpose in the past for performance reasons to not iterate over the whole enumerable.

Reply:
`Any()` vs `Count()` or vs `Count`?
Using `Count` (property) is [orders of magnitude](https://www.tabsoverspaces.com/233649-comparing-speed-of-count-0-and-any) faster than using any of the LINQ methods and doesn't iterate over the whole enumerable.
You can argue about readability though.

Reply:
Ah, so `Count` is static. Then let's use that if LINQ is slower.